Hypnotic-Herons/bangazon-api
# added product type to db

## Link to Ticket
Closes  https://github.com/Hypnotic-Herons/bangazon-api/issues/5


## Description of Proposed Changes
Added files to create a table for product type in the db and manage that data
- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout dvProductType
code .

```


1. makemigrations and migrate
1. run server
1. view in browser to see the api root
1. add /product_type to the browser url and see the detail view for product type instances - you should see "category" listed for the entry form on the bottom of the page

1. open db browser and see the table product_type




  

## Impacted Areas in Application

List general components of the application that this PR will affect:

* data structure for the project

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
